src/translations/fr.json

```diff
@@ -269,7 +269,7 @@
     "sensitivityAnalysisError": "Une erreur est survenue lors de l'exécution de l'analyse de sensibilité",
     "SensitivityInDeltaMW": "Sensibilité au niveau des quadripôles en ΔMW",
     "SensitivityInDeltaA": "Sensibilité au niveau des quadripôles en ΔA",
-    "SensitivityAtNode": "Sensibilité au niveau des noeuds",
+    "SensitivityAtNode": "Sensibilité au niveau des nœuds",
```

Comment:
why this change ?

Reply:
[linguistic rule](https://fr.wikipedia.org/wiki/%C5%92#Emplois_modernes) in french